CHANGELOG.md

```diff
@@ -6,6 +6,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.5.0] - Sep, 22, 2021
+
+### Core
+- [Text Input] Added Info message to inform user of dynamic input value changes
+- [Icons] New icons:
+  - Shopping cart
+
+### React Components
+
+#### Added
+- File Input
+- [Text Input, TextArea, Password Input, Number Input, Time Input, Date Input, File Input] Added Info message to inform user of dynamic input value changes
+- [Icons] New icons:
+  - Shopping cart
+
+#### Change
+- [Accordion] Changed Accordion to open via header click
+- [Accordion] Added support for accordion being open by default
+- Use DateInput and Phone Input in the form examples
```

Comment:
Phone Input is also validated, though not very strictly. It only requires the international mobile number format. Maybe no need to mention that separately

Reply:
- [ ] add phone input validation text